packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -450,6 +456,7 @@ export const startRewardDistributor = async ({
     Object.entries(collectorKit).map(async ([debugName, collectorFacet]) => {
       const collector = E(instanceKit.creatorFacet).makeContractFeeCollector(
         zoe,
+        // @ts-expect-error Property 'makeCollectFeesInvitation' is missing in type 'StakeFactoryCreator' but required in type 'CollectibleContractFacet
```

Comment:
finding that error seems like a feature

Reply:
governance improvements in master that landed right after I pushed fixed this. that's why I force pushed, but unfortunately that change didn't go along. I'll push again.